internal/plugin/grpc_client_plugin_test.go

```diff
@@ -17,6 +26,8 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+var grpcServerMutex = &sync.Mutex{}
```

Comment:
nit: https://github.com/uber-go/guide/blob/master/style.md#zero-value-mutexes-are-valid
```suggestion
var grpcServerMutex = sync.Mutex{}
```

Reply:
removed this code

Reply:
looking at the code, probably need a task to address this across the board